test/unit/master/test_build_request_handler.py

```diff
@@ -0,0 +1,40 @@
+from genty import genty, genty_dataset
+from unittest.mock import Mock
+
+from app.project_type.project_type import ProjectType
+from app.master.atom import Atom
+from app.master.build_request_handler import BuildRequestHandler
+from app.master.atomizer import Atomizer
+from app.master.job_config import JobConfig
+from test.framework.base_unit_test_case import BaseUnitTestCase
+
+
+@genty
+class TestBuildRequestHandler(BaseUnitTestCase):
+    @genty_dataset(
+        atoms_override_specified=(['override1', 'override2'], None, False),
+        atoms_override_not_specified=(None, [Atom('atom_value_1'), Atom('atom_value_2')], True),
+    )
+    def test_compute_subjobs_for_build(self, atoms_override, atomizer_output, atomizer_called):
```

Comment:
Right, if you wanted to be totally specific you could use "...if_and_only_if...". My main point was that without a behavior in the test name it feels like it's saying we're testing literally everything the `compute_subjobs_for_build` method does.